examples/functions/timetable/Netex_52_planned_delay.xml

```diff
@@ -64,36 +64,37 @@
 					</scheduledStopPoints>
 					<notices>
 						<Notice id="delay_notice" version="any">
-						<Text>During July the bus is delayed by 5 minutes due to construction work.</Text>
+							<Text>During July the bus is delayed by 5 minutes due to construction work.</Text>
 						</Notice>
 					</notices>
-					</ServiceFrame>
+				</ServiceFrame>
 				<!-- ======TIMETABLE ======= -->
 				<TimetableFrame version="1.0" id="ch:tf">
 					<validityConditions>
 						<AvailabilityCondition version="any" id="all_year_no_longer_available">
 							<Description>regular run</Description>
 							<FromDate>2025-11-01T00:00:00Z</FromDate>
 							<ToDate>2025-12-31T00:00:00Z</ToDate>
-							<IsAvailable>false</IsAvailable>  <!--making clear that it is no longer available -->
+							<IsAvailable>false</IsAvailable>
+							<!--making clear that it is no longer available -->
 						</AvailabilityCondition>
 						<AvailabilityCondition version="any" id="only_july">
 							<Description>regular run</Description>
 							<FromDate>2025-07-01T00:00:00Z</FromDate>
 							<ToDate>2025-07-31T23:59:00Z</ToDate>
-						</AvailabilityCondition>	
+						</AvailabilityCondition>
 						<AvailabilityCondition version="any" id="all_except_july">
 							<Description>regular run</Description>
 							<FromDate>2025-07-01T00:00:00Z</FromDate>
 							<ToDate>2025-07-31T23:59:00Z</ToDate>
-						</AvailabilityCondition>						
+						</AvailabilityCondition>
 					</validityConditions>
 					<vehicleJourneys>
 						<ServiceJourney version="any" id="444">
-						<validityConditions>
-							<ValidityConditionRef ref="all_year_no_longer_available" version="any"/>
-						</validityConditions>
-						<Description>The original journey replaced by 444-1 and 444-2</Description>
+							<validityConditions>
+								<ValidityConditionRef ref="all_year_no_longer_available" version="any"/>
```

Comment:
Please use specific references, hence AvailabilityConditionRef.

Reply:
done (you could just do it yourself)